### ISSUE TITLE
[Question] Ways to address accessibility?

### ISSUE THREAD

**Issue by shawngraham** [rstudio/bookdown#335]
Hi,

Bookdown is exactly what I need for a project I'm working on. However, the final product, whether website, pdf, or epub, has to be compliant with accessibility standards (things like use by screen-readers or other accessibility techs). I wondered what the consensus was on best ways to address accessibility concerns. Would the most elegant approach be to address this through the css or in the source templates? see for example [http://wave.webaim.org/report#/https://bookdown.org/yihui/bookdown/](http://wave.webaim.org/report#/https://bookdown.org/yihui/bookdown/) for a report on accessibility of the 'getting started' book. 

Thanks,
Shawn

**Comment by yihui** [comment]
I see two issues there. One is about the missing alt text for the cover image, and one is about the missing language specification. The latter can be fixed by adding a `lang` field in the YAML of index.Rmd. The former is not really important. The reader won't miss any important information without realizing there is a cover image on the page. This issue is specific to this image in the bookdown book.

Thanks for bringing this to our attention! If you notice any general issues about bookdown, I'd love to fix them.

**Comment by shawngraham** [comment]
Thank you!
